### PR TITLE
Add Discord Link To Sidebar

### PR DESCRIPTION
**Pull request by thebeanogamer**

### CODE REVIEW

File: lib/pages/home_page.dart [hunk 1]
```diff
@@ -79,6 +81,11 @@ class _HomePageState extends State<HomePage> {
                     Navigator.pop(context);
                   });
                 }),
+                new ListItem(Icons.group, "Discord", (){setState(
+                        (){
```
Comment (REDACTED-REDACTED) [inline lib/pages/home_page.dart]: This indentation is triggering me

**Comment by thebeanogamer** [comment]
@REDACTED-REDACTED I have made the requested changes.

File: .gitignore [hunk 2]
```diff
@@ -1,12 +1,260 @@
...
+bin/
+gen/
+out/
+
+# Gradle files
+.gradle
+.gradle/
 build/
-.idea/
-.vscode/
```
Comment (REDACTED-REDACTED) [inline .gitignore]: still not ignoring `.vscode`

Comment (thebeanogamer) [reply .gitignore]: `.vscode` is removed on line 234
`.idea` is removed on lines 68-88

Comment (REDACTED-REDACTED) [reply .gitignore]: Yeah to ignore it is need to be in the gitignore

Comment (thebeanogamer) [reply .gitignore]: I understand that. Those folders are ignored due to the lines I mentioned above.

File: .gitignore [hunk 3]
```diff
@@ -1,12 +1,260 @@
...
+.idea/scopes/scope_settings.xml
+.idea/dictionaries
+.idea/vcs.xml
+.idea/jsLibraryMappings.xml
+.idea/datasources.xml
+.idea/dataSources.ids
+.idea/sqlDataSources.xml
+.idea/dynamic.xml
+.idea/uiDesigner.xml
+.idea/codeStyles/Project.xml
```
Comment (REDACTED-REDACTED) [inline .gitignore]: This doesn't cover all the local configuration file can be replaced with `.idea/` as it was before

**Comment by thebeanogamer** [comment]
Whilst I don't think it is needed, I have made the changes @REDACTED-REDACTED suggested.

**Comment by thebeanogamer** [comment]
This PR has been abandoned due to commiting in the wrong order.

**Pull request merged**